tests/replication/tcp/streams/test_typing.py

```diff
@@ -99,75 +100,83 @@ def test_reset(self) -> None:
         This is emulated by jumping the stream ahead, then reconnecting (which
         sends the proper position and RDATA).
         """
-        typing = self.hs.get_typing_handler()
-        assert isinstance(typing, TypingWriterHandler)
-
-        # Create a typing update before we reconnect so that there is a missing
-        # update to fetch.
-        typing._push_update(member=RoomMember(ROOM_ID, USER_ID), typing=True)
-
-        self.reconnect()
-
-        typing._push_update(member=RoomMember(ROOM_ID, USER_ID), typing=True)
-
-        self.reactor.advance(0)
-
-        # We should now see an attempt to connect to the master
-        request = self.handle_http_replication_attempt()
-        self.assert_request_is_get_repl_stream_updates(request, "typing")
-
-        self.mock_handler.on_rdata.assert_called_once()
-        stream_name, _, token, rdata_rows = self.mock_handler.on_rdata.call_args[0]
-        self.assertEqual(stream_name, "typing")
-        self.assertEqual(1, len(rdata_rows))
-        row: TypingStream.TypingStreamRow = rdata_rows[0]
-        self.assertEqual(ROOM_ID, row.room_id)
-        self.assertEqual([USER_ID], row.user_ids)
-
-        # Push the stream forward a bunch so it can be reset.
-        for i in range(100):
-            typing._push_update(
-                member=RoomMember(ROOM_ID, "@test%s:blue" % i), typing=True
+        # A huge RDATA log line is triggered in this test, which breaks trial
+        # ref: https://github.com/twisted/twisted/issues/12482
```

Comment:
Alternative to actually call this out as a FIXME.

```suggestion
        # FIXME: Because we emit a huge RDATA log line and trial sometimes crashes on
        # these, disable that logger for this test. This can be removed once
        # https://github.com/twisted/twisted/issues/12482 is solved and we update
        # Twisted to version that includes the fix.
```

(applies to other spot as well)

Reply:
(guess I should add that caveat to the comment though)